.github/workflows/website.yml

```diff
@@ -37,6 +37,7 @@ jobs:
       - uses: bahmutov/npm-install@v1
       - run: npm run build -w packages/client
       - run: npm run build -w packages/website
+      - run: npm run build -w packages/docs
```

Comment:
the `env` property on the next line has been detached from the the run step that needs it above this line